testdata/project-v2/config/crd/bases/crew.testproject.org_admirals.yaml

```diff
@@ -9,9 +9,7 @@ spec:
   group: crew.testproject.org
   names:
     kind: Admiral
-    listKind: AdmiralList
```

Comment:
This change is unrelated.
I guess it's caused by not using the latest controller-tools.

Reply:
Hi @mengqiy, 

I think it is some PR merged without update the testdata, because I did;
- rebase with the master
- `go clean --modcache` and `go clean`
- make and make install 
- the executed the script generated_golden.sh

And these files still changed. 
Please, let me know if I misunderstand something here or has anything else that I should do.

Reply:
There is an existing issue that controller-gen doesn't automatically upgrade.
To ensure it's the latest, you can do the following:
- which controller-gen
- rm -f /path/to/controller-gen
- run `which controller-gen` to ensure it's gone and there is only on copy in your PATH
- GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.1
- run `which controller-gen` to confirm it has been successfully re-installed.

After that re-run the script.

Reply:
Worked 👍